docs/platforms/react-native/manual-setup/metro.mdx

```diff
@@ -79,3 +79,4 @@ export type Bundle = {
 
 - Sentry Metro Serializer can't add Debug ID to the Hermes Composed Source Maps. Please see [Manual Upload With Hermes](/platforms/react-native/sourcemaps#manual-upload-with-hermes) guide on how to add Debug ID to the Hermes Composed Source Maps.
 - If you see `Debug ID was not found in the bundle.` error message the `sentryBundleCallback` was not called by your custom serializer.
+- Metro configuration is optional in Expo projects, to create it call `npx expo customize metro.config.js`. Read more about [Customizing Expo Metro Configuration](https://docs.expo.dev/guides/customizing-metro/).
```

Comment:
`m`: can we add a note about when/why users would want to add custom metro config?